src/platform/utilities/ui/index.js

```diff
@@ -34,7 +34,7 @@ export function focusElement(selectorOrElement, options) {
 
 // Allows smooth scrolling to be overridden by our E2E tests
 export function getScrollOptions(additionalOptions) {
-  const globals = window.VetsGov || {};
+  const globals = window.Forms || {};
```

Comment:
This change is still a bit concerning. There are lots of tests that are setting `window.VetsGov` with an expectation that that will have an impact on behavior, and if the code no longer uses that `VetsGov` property then the expectation may not happen. Even if tests don't break, the tests may be giving false positives.

Reply:
This actually is not changing how those operate, though. This change cited here is inside the `getScrollOptions` function. The other applications that were changed to source off this function, were all referencing the `scroll` property....whether that be `VetsGov.scroll` or `Forms.scroll`.  I've inserted a screenshot from `mocha-setup.js` where those scroll options are defined and they are identical. The other places `VetsGov` is still referenced will be unaffected because the rest of that object has not been altered or changed. The only thing I am dealing with in this PR is scroll-related.  Because the below has been left untouched, that variable would still exist there if there's a reference to it somewhere that needs the scroll options there (however, I do not see one in the global file search).

![image](https://user-images.githubusercontent.com/34250637/134531614-2dc51694-a2bc-4831-8135-bcac846369f2.png)

Reply:
I see `window.VetsGov.scroll` in some other files, but I suppose those can be dealt with in another PR.

![image](https://user-images.githubusercontent.com/2008881/135169580-5be2a416-2f37-4871-b919-818574c9a736.png)